tests/GraphBLAS-sharp.Tests/paket.references

```diff
@@ -8,3 +8,5 @@ MathNet.Numerics.FSharp
 
 Brahma.FSharp
 TypeShape
+
+YC.QuickGraph
```

Comment:
It may be better to use [QuikGraph](https://www.nuget.org/packages/QuikGraph) instead. YC.QuickGraph looks dead.